.travis.yml

```diff
@@ -46,7 +58,7 @@ jobs:
         - mysql
         - redis-server
       before_script:
-        - mysql -u root -e "DROP DATABASE IF EXISTS superset; CREATE DATABASE superset DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci"
+        - mysql -u root -e "DROP DATABASE IF EXISTS superset; CREATE DATABASE superset DEFAULT CHARACTER SET utf8"
```

Comment:
@michellethomas this was a manual override to make the MySQL tests past as it was appending this to the column types which violated the column size condition. I'm uncertain why this is occurring after the Travis upgrade as we haven't seen it on `apache/incubator-superset`.